### DISCUSSION TITLE
Primer benchmark

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#31] [General]
A small issue for you to look at ;)

When running the pcr primer benchmark for the "meat" primers against the current version of Genbank nt eukaryotes (see: https://ftp.ncbi.nlm.nih.gov/blast/db/), the taxidtools stage dies on a taxid, 666121. This id is only included in merged.dmp, not nodes.dmp. The benchmark workflow uses an older version of taxidtools that doesn't know about merged.dmp and fails rather catastrophically. I suppose this could either be caught and ignored, or the code could be updated to taxidtools >= 3.0. But since Snakemake isn't exactly my area of expertise, I am stuck for the moment.

**Comment by gregdenay** [comment]
Wrong repo :)
I'll push an update to the barcode pipeline
